samples/iothub-connect-to-eventhubs/README.md

```diff
@@ -0,0 +1,39 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-iot-hub
+- azure-event-hubs
+urlFragment: iothub-connect-to-eventhubs
+name: Request the IoT Hub built-in Event Hubs-compatible endpoint connection string
+description: This sample illustrates the flow of querying an IoT instance to obtain a connection string that can be used with the Event Hub instance to which IoT Hub is currently publishing as it's "built-in" endpoint.
+---
+# Request the IoT Hub built-in Event Hubs-compatible endpoint connection string
+
+IoT Hub is a managed service that acts as a central message hub for bi-directional communication between an IoT application and the devices it manages. IoT Hub supports communications both from the device to the cloud and from the cloud to the device. IoT Hub supports multiple messaging patterns and may be integrated with other Azure services to build complete, end-to-end solutions.
+
+A common pattern is for users of IoT Hub to consume information from an Event Hub instance that is provisioned and owned by the IoT Hub service. The [IoT Hub documentation](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-endpoints) refers to this Event Hub instance as the "built-in Event Hub-compatible endpoint that IoT Hub exposes", making it an important part of the IoT Hub story.  One of the features supported by IoT Hub is a fail-over scenario, which may be initiated by customers, and which is likely to result in IoT Hub changing the Event Hub instance to which it publishes events. Because this Event Hub instance association is owned by IoT Hub, developers cannot assume a static connection string to access it, but instead must query the IoT Hub service for the active instance.
+
+This sample illustrates querying the IoT Hub service to obtain the connection string of the Event Hub instance to which IoT Hub is currently publishing as its "built-in" endpoint.
+
+## Getting started
+
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
```

Comment:
Thanks.  This blurb seems to have gotten stale since first written.  Making the change here and I'll apply to the other Event Hubs READMEs in a separate PR.